src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -383,6 +376,7 @@ protected bool BuildInternal (string projectOrSolution, string target, string []
 							stdout.Set ();
 					};
 					p.StartInfo = psi;
+					Console.WriteLine ($"{psi.FileName} {psi.Arguments}");
```

Comment:
Is this already the first line of the `process.log`?

https://github.com/xamarin/xamarin-android/blob/d298fdaadab1a80bc45f9334d946c0ae8b568c42/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs#L356-L357

Do we need it twice? The output on Azure DevOps now looks like:
```
C:\A\vs2019xam00000R-1\_work\2\s\bin\Release\bin\xabuild.exe C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_True_False\Lib1\Lib1.csproj /t:Clean /noconsolelogger "/flp1:LogFile=C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_True_False\Lib1\build.log;Encoding=UTF-8;Verbosity=diagnostic" @"C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_True_False\Lib1\project.rsp"
Found Time Elapsed 00:00:01.0300000
C:\A\vs2019xam00000R-1\_work\2\s\bin\Release\bin\xabuild.exe C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_True_False\Lib1\Lib1.csproj /t:Build /noconsolelogger "/flp1:LogFile=C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_True_False\Lib1\build.log;Encoding=UTF-8;Verbosity=diagnostic" @"C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_True_False\Lib1\project.rsp"
Found Time Elapsed 00:00:01.9700000
C:\A\vs2019xam00000R-1\_work\2\s\bin\Release\bin\xabuild.exe C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_True_False\UnnamedProject\UnnamedProject.csproj /t:Compile /noconsolelogger "/flp1:LogFile=C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_True_False\UnnamedProject\build.log;Encoding=UTF-8;Verbosity=diagnostic" @"C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_True_False\UnnamedProject\project.rsp"
Found Time Elapsed 00:00:01.4500000
C:\A\vs2019xam00000R-1\_work\2\s\bin\Release\bin\xabuild.exe C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_True_False\UnnamedProject\UnnamedProject.csproj /t:Build /noconsolelogger "/flp1:LogFile=C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_True_False\UnnamedProject\build.log;Encoding=UTF-8;Verbosity=diagnostic" @"C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_True_False\UnnamedProject\project.rsp"
Found Time Elapsed 00:00:12.9200000
[TESTLOG] Test DesignTimeBuildTrueTrueFalse Complete
[TESTLOG] Test DesignTimeBuildTrueTrueFalse Outcome=Passed
=> Xamarin.Android.Build.Tests.AndroidUpdateResourcesTest.DesignTimeBuild(True,False,True)
[TESTLOG] Test DesignTimeBuildTrueFalseTrue Starting
C:\A\vs2019xam00000R-1\_work\2\s\bin\Release\bin\xabuild.exe C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_False_True\Lib1\Lib1.csproj /t:Clean /noconsolelogger "/flp1:LogFile=C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_False_True\Lib1\build.log;Encoding=UTF-8;Verbosity=diagnostic" @"C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_False_True\Lib1\project.rsp"
Found Time Elapsed 00:00:00.4200000
C:\A\vs2019xam00000R-1\_work\2\s\bin\Release\bin\xabuild.exe C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_False_True\Lib1\Lib1.csproj /t:Build /noconsolelogger "/flp1:LogFile=C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_False_True\Lib1\build.log;Encoding=UTF-8;Verbosity=diagnostic" @"C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_False_True\Lib1\project.rsp"
Found Time Elapsed 00:00:02.9900000
C:\A\vs2019xam00000R-1\_work\2\s\bin\Release\bin\xabuild.exe C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_False_True\UnnamedProject\UnnamedProject.csproj /t:Compile /noconsolelogger "/flp1:LogFile=C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_False_True\UnnamedProject\build.log;Encoding=UTF-8;Verbosity=diagnostic" @"C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_False_True\UnnamedProject\project.rsp"
Found Time Elapsed 00:00:02.6300000
C:\A\vs2019xam00000R-1\_work\2\s\bin\Release\bin\xabuild.exe C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_False_True\UnnamedProject\UnnamedProject.csproj /t:Build /noconsolelogger "/flp1:LogFile=C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_False_True\UnnamedProject\build.log;Encoding=UTF-8;Verbosity=diagnostic" @"C:\A\vs2019xam00000R-1\_work\2\s\bin\TestRelease\temp\DesignTimeBuild_True_False_True\UnnamedProject\project.rsp"
Found Time Elapsed 00:00:14.8200000
[TESTLOG] Test DesignTimeBuildTrueFalseTrue Complete
[TESTLOG] Test DesignTimeBuildTrueFalseTrue Outcome=Passed
```

Reply:
Oh yeah I forgot about this, maybe we don't need the extra process log file at all anymore?

Reply:
I think the `process.log` would show `nuget.exe` output. There are still some tests that use `packages.config`.

It might also be possible for MSBuild to crash, and the output might only be in `process.log`.

Reply:
@jonathanpeppers I could go either way on this one. The process log has some more data and includes the response file arguments, so to remove duplication it probably makes more sense to remove the new Console.WriteLine that I added. Alternatively we could just keep some duplication here since it can be nice to get a better idea of what's running in real time on CI.

Reply:
I think it's fine, it can't hurt to print this.